docs/modules/ROOT/pages/explanation/synthesize.adoc

```diff
@@ -0,0 +1,144 @@
+= Synthesize a cluster
+
+include::partial$synthesize-macros.adoc[]
+
+This explanation goes into the details that are omitted in xref:how-tos/synthesize.adoc[synthesizing a cluster].
+
+== Creating a new cluster on control.vshn.net
+
+There's quite a few fields to fill out when creating a cluster.
+
+ID:: Choose a unique ID for the cluster.
+This field is limited to lower-case characters `a-z`, numbers `0-9` and `-`.
+The length is limited to 1-61 characters.
++
+NOTE: This field cannot be changed after creation, it's used as the Kubernetes object name for the cluster by Lieutenant.
++
+NOTE: We're planning to make the ID field optional, see ticket https://ticket.vshn.net/browse/PORTAL-49[PORTAL-49] in Jira.
+Display Name:: Choose a descriptive name.
+Check first on the cluster listing overview how other clusters of the tenant are named.
+Lieutenant Tenant:: Choose the tenant to which this cluster belongs
+Service Level (Fact: service_level):: Check with Service Manager which Service Level applies to this cluster
+Cloud":: Check folder `cloud` in the {commodore-defaults-repo} for already existing values for cloud.
+If you set this field to `<name>`, there must be a file `cloud/<name>.yml` in the {commodore-defaults-repo}.
++
+Use value `none` for on-premise setups.
++
+See the section on <<_creating_a_missing_cloud,creating a missing cloud>> for some notes on adding a new cloud.
+"Region":: Assuming that the value of field "Cloud" is `<cloudname>`, check folder `cloud/<cloudname>` in the {commodore-defaults-repo} for already existing regions for the cloud.
++
+Field "Region" is not required to be set for "Cloud" `none`.
+"Distribution":: Check folder `distribution` in the {commodore-defaults-repo} for already available distributions.
+
+Rancher Cluster ID (Fact: rancher_cluster_id):: If it's a Rancher cluster, add the cluster ID from the Rancher Management Server to this field
+Repository Type":: Leave at "auto" to have Lieutenant manage the cluster catalog repo on https://git.vshn.net
+GitRepo URL":: Leave empty if "Repository Type" is auto.
+Check with the {projectsyn-ig}, if you have special requirements for the cluster catalog repo.
+
+[NOTE]
+====
+Not all combinations of attributes work smoothly.
+If in doubt, seek help from the {projectsyn-ig}.
+====
+
+=== Creating a missing cloud
```

Comment:
Maybe move these three sections to their own how-tos?

Reply:
done